galaxykit/client.py

```diff
@@ -73,15 +84,15 @@ def _http(self, method, path, *args, **kwargs):
         headers = kwargs.pop("headers", self.headers)
         parse_json = kwargs.pop("parse_json", True)
 
-        resp = requests.request(method, url, headers=headers, *args, **kwargs)
+        resp = requests.request(method, url, headers=headers, verify=False, *args, **kwargs)
```

Comment:
What's the reasoning behind turning `verify` to False in this spot? Not challenging, just curious about the purpose. I think it disables SSL verification?

Reply:
yes, but i'll make it an option. it makes sense for tests, where all our test instances have self-signed certs.

Reply:
OK, cool - just curious. I agree it does make sense for most our uses.